cloudblue_connect.custom_adapter.rb

```diff
@@ -0,0 +1,694 @@
+      "external_id"=>"#{input["external_id"]}",
+      "items"=> input["product_items"],
+      "params"=>[],
+      "tiers"=>{
+         "customer"=>{
+            "name"=>"#{input["customer_account_name"]}",  
+            "type"=>"customer",        
+            "external_id"=>"#{input["customer_external_id"]}",  
+            "external_uid"=>"#{input["customer_external_uid"]}",  
+        
+            "contact_info"=>{
+               "address_line1"=>"#{input["customer_address_line1"]}",    
+               "address_line2"=>"#{input["customer_address_line2"]}",
+               "city"=>"#{input["customer_city"]}",
+               "state"=>"#{input["customer_state"]}",
+               "postal_code"=>"#{input["customer_postal_code"]}",
+               "country"=>"#{input["customer_country"]}",
+               "contact"=>{
+                  "first_name"=>"#{input["customer_contact_first_name"]}",     
+                  "last_name"=>"#{input["customer_contact_last_name"]}",    
+                  "email"=>"#{input["customer_email"]}",    
+                  "phone_number"=>{
+                     "country_code"=>"",
+                     "area_code"=>"",
+                     "phone_number"=>"",
+                     "extension"=>""
+                  }
+               }
+            }
+         },
+         "tier1"=>{
+            "name"=>"#{input["tier1_account_name"]}", 
+            "type"=>"reseller",
+            "external_uid"=>"3444",  
+
+          "contact_info"=>{
+               "address_line1"=>"#{input["tier1_address_line1"]}",    
+               "address_line2"=>"#{input["tier1_address_line2"]}",
+               "city"=>"#{input["tier1_city"]}",
+               "state"=>"#{input["tier1_state"]}",
+               "postal_code"=>"#{input["tier1_postal_code"]}",
+               "country"=>"#{input["tier1_country"]}",
+               "contact"=>{
+                  "first_name"=>"#{input["tier1_first_name"]}",     
+                  "last_name"=>"#{input["tier1_last_name"]}",    
+                  "email"=>"#{input["tier1_email"]}",    
+                  "phone_number"=>{
+                     "country_code"=>"",
+                     "area_code"=>"",
+                     "phone_number"=>"",
+                     "extension"=>""
+                  }
+               }
+            }
+           
+         }
+      }
+   },
+   "marketplace"=>{
+      "id"=>"MP-90040"
+   }
+  }  
+        post("https://api.connect.cloudblue.com/public/v1/requests", hardcoded_params).
+          after_error_response(/.*/) do |_, body, _, message|
+            error("#{message}: #{body}")
+          end
+      end,
+      
+      output_fields: lambda do |object_definitions|
+        object_definitions["purchase_request"]
+      end
+    },
+        
+    
+    
+    change_items_quantity: {
+      title: "Change items quantity",
+      subtitle: "",
+      description: "",
+      help: "",
```

Comment:
For sake of consistency, this action must be "Asset change request"
Also need to add help link